src/styles/reset.css

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
```

Comment:
all of these are normalize styles, so move these to `normalize.css` you can leave margin reset here tho